### ISSUE TITLE
Add Open Graph markup

### ISSUE THREAD

**Issue by jqnatividad** [talos/opendatacache#5]
So that datasets will be Discourse onebox-eable

**Comment by talos** [comment]
Good idea.  Would you be able to contribute a PR?  Codebase should be pretty approachable -- all changes would be made on `index.html`.

**Comment by jqnatividad** [comment]
Will do for this and #4 as well....

**Comment by talos** [comment]
Thank you!

**Comment by jqnatividad** [comment]
Taking longer than expected since both SEO mark-ups are not really optimized for dynamic, javascript-driven sites, though schema.org does support it with JSON-LD syntax (https://developers.google.com/structured-data/schema-org)

Bigger problem is there is no URL to link to even if its indexed since there's no permalink for each dataset...

**Comment by talos** [comment]
Not sure what you mean -- each dataset has a permalink.  For example:

```
http://www.opendatacache.com/bronx.lehman.cuny.edu/api/views/228m-qr7g/rows.csv
```
